woocommerce/woocommerce
# Update workflows to run build zip script

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes issue with 4 of our GitHub action workflows. `build-release-zip-file.yml`, `nightly-builds.yml`, `smoke-test-daily.yml` and `build-release.yml`. Mainly it was because `makepot` script was failing and by adding specific PHP version, it resolve the issue.

### How to test the changes in this Pull Request:

1. Checked that the tests are passing below.
2. Manually trigger the `build release zip` action by going to here https://github.com/woocommerce/woocommerce/actions/workflows/build-release-zip-file.yml
3. Click on `Run workflow` and choose this branch `fix/workflows-3` to target and run it.
4. Ensure the action runs successfully.

### Other information:




### FOR PR REVIEWER ONLY:
